faunadb-common/src/main/java/com/faunadb/common/Connection.java

```diff
@@ -455,9 +477,17 @@ private HttpRequest makeHttpRequest(String httpMethod, String path, Optional<Jso
           X_FAUNADB_API_VERSION, API_VERSION,
           USER_AGENT, userAgent,
           X_FAUNA_DRIVER, jvmDriver.toString(),
+          X_FAUNA_DRIVER_VERSION, Connection.class.getPackage().getSpecificationVersion(),
```

Comment:
This code is executed for each requests, I think it would make sense to pre-compute once all the necessary values to reuse them.

Reply:
Good catch!